bot/__init__.py

```diff
@@ -64,6 +64,7 @@ def monkeypatch_trace(self: logging.Logger, msg: str, *args, **kwargs) -> None:
 logging.getLogger("discord").setLevel(logging.WARNING)
 logging.getLogger("websockets").setLevel(logging.WARNING)
 logging.getLogger("chardet").setLevel(logging.WARNING)
+logging.getLogger("async_rediscache").setLevel(logging.WARNING)
 logging.getLogger(__name__)
```

Comment:
Does this call have any use? Didn't want to remove it in case I missed something

Reply:
Good question. I doubt it's useful, but it may actually do something (as in, create a logger instance there in the logging hierarchy instead of a placeholder object). Not sure why it's there, honestly.

Reply:
I think it can be removed. The line was added in ebead84ae0eb62ab64b329dfd7fff61ff5de628d and left in this state after e8ad8bb3cf735e9fef09413b8b118b2eff601797.